ADA/databricks_rstudio.qmd

```diff
@@ -65,29 +65,40 @@ You must have:
+        d.  Click the 'OK' button to exit the 'Test Results' window, then the 'OK' button in the 'Simba Spark ODBC Driver DSN Setup' window.
 
-5.  Connect through RStudio. Now the connection between our laptop and Databricks works we can use it to query data stored in Databricks from RStudio.
+5.  Connect through RStudio. Watch the below video and view the [ADA_RStudio_connect GitHub repo](https://github.com/dfe-analytical-services/ADA_RStudio_connect) for methods on connecting to Databricks and querying data from RStudio. 
 
-    i)  Open RStudio, install the 'RODBC' package and load the 'RODBC' library.
 
-    ii) Create a connection variable by passing the 'Data Source Name' you gave to your ODBC connection to the odbcConnect() function. `conn <- obbcConnect("DatabricksSQL Warehouse")`
+------------------------------------------------------------------------
+
+# Pulling data into R studio from Databricks
+
+------------------------------------------------------------------------
+
+Once you have set up an ODBC connection as detailed above, you can then use that connection to pull data directly from Databricks into R Studio. Charlotte recorded a video demonstrating two possible methods of how to do this. The recording is embedded below: 
+
+<div align="center">
+<iframe src="https://educationgovuk.sharepoint.com/sites/lvewp00086/_layouts/15/embed.aspx?UniqueId=5fad039e-a763-40c8-8ea0-403aea712f4c&embed=%7B%22ust%22%3Atrue%2C%22hv%22%3A%22CopyEmbedCode%22%7D&referrer=StreamWebApp&referrerScenario=EmbedDialog.Create" width="640" height="360" frameborder="0" scrolling="no" allowfullscreen title="ADA_Rstudio_2.mp4"></iframe>
+</div>
+
+A template of all of the code used in the above video can be found in the [ADA_RStudio_connect GitHub repo](https://github.com/dfe-analytical-services/ADA_RStudio_connect).
 
-6.  Query Data. You can now use your connection variable to send SQL queries to Databricks.
+Key takeaways from the video and example code: 
 
-    i)  Use the `sqlQuery()` function with the connection variable to see what data catalogues are available on DataBricks *Databricks uses the American spelling of 'catalog'.* ![](../images/databricks-catalogs-rstudio.png)
-    ii) Use the `sqlQuery(conn, "USE CATALOG preprod_catalog_p02")` command to tell Databricks which catalogue to use. *Ensure that you have the correct permissions for the catalogue first. If you don't you will need to contact the ADA team.*
-    iii) You can now send SQL queries like you would using any other database. To see information about all the tables you have access to you can query the INFORMATION_SCHEMA using the `"SELECT * FROM INFORMATION_SCHEMA.TABLES"` query like so;
+* The main change here is the connection method. The installation and setup of the ODBC driver are all done pre-code, and the only part of the code that will need updating is your connection (usually your con variable). 
```

Comment:
Maybe just tweak this to say something like "the main change from interacting with a non-ADA hosted SQL database is the connection method", if that sounds right?